src/include/redismodule.h

```diff
@@ -4,6 +4,29 @@
 #include <sys/types.h>
 #include <stdint.h>
 #include <stdio.h>
+#include <stdlib.h>
```

Comment:
Did you took this file from unstable or from Redis 7.0 branch?

Reply:
fixed